packages/mui-material/src/StepLabel/StepLabel.test.js

```diff
@@ -123,6 +134,17 @@ describe('<StepLabel />', () => {
   });
 
   describe('<Step /> prop: completed', () => {
+    it('renders <StepLabel> with the className completed', () => {
+      const { container } = render(
+        <Step completed>
+          <StepLabel>Step One</StepLabel>
+        </Step>,
+      );
+
+      const root = container.querySelector(`.${classes.root}`);
+      expect(root).to.have.class(classes.active);
```

Comment:
This test should have failed. Because it should be:
```suggestion
      expect(root).to.have.class(classes.completed);
```